src/ElasticaBundle/DependencyInjection/ElasticaExtension.php

```diff
@@ -84,8 +98,37 @@ private function loadClient($clientName, array $clientConfig, ContainerBuilder $
                 'log',
                 $container->getParameter('kernel.debug')
             ])
-            ->setPublic(true)
-        ;
+            ->setPublic(true);
+    }
+
+    /**
+     * Configure service auto-wiring for default Elastica client
+     * for Symfony 3.3+
+     *
+     * @param array $config
+     * @param ContainerBuilder $container
+     * @throws \Symfony\Component\DependencyInjection\Exception\InvalidArgumentException
```

Comment:
Please, don't use the `@throws` annotation for exceptions that are not explicitly thrown by the method

Reply:
Removed, however personally I think that it is a good practice